e2e/testcases/apply_scoped_resource_test.go

```diff
@@ -1,138 +0,0 @@
-// Copyright 2022 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package e2e
-
-import (
-	"testing"
-	"time"
-
-	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
-	"kpt.dev/configsync/e2e/nomostest"
-	"kpt.dev/configsync/e2e/nomostest/kubevirt/v1"
-	"kpt.dev/configsync/e2e/nomostest/metrics"
-	"kpt.dev/configsync/e2e/nomostest/ntopts"
-	"kpt.dev/configsync/e2e/nomostest/retry"
-	nomostesting "kpt.dev/configsync/e2e/nomostest/testing"
-	"kpt.dev/configsync/pkg/api/configsync"
-)
-
-func TestApplyScopedResources(t *testing.T) {
```

Comment:
It seems like we should be capturing the spirit of this test with a different (simpler) resource type rather than removing this test completely

Reply:
@karlkfi , @sdowell , https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/e2e/testcases/gatekeeper_test.go tests having a `ConstraintTemplate` CR and a `K8sAllowedRepos` CR in the SOT. However, the test does not run a gatekeeper controller, and simulate the Gatekeeper's controller behavior by applying the `k8sallowedrepos.constraints.gatekeeper.sh` CRD with kubectl: https://github.com/GoogleContainerTools/kpt-config-sync/blob/f7c498ae97013c726cf50ae302d8ff60b2566c46/e2e/testcases/gatekeeper_test.go#L86-L94.

Reply:
Thinking more about this, simulating the Gatekeeper's controller behavior by applying the k8sallowedrepos.constraints.gatekeeper.sh CRD with kubectl is enough for testing purpose.